server/src/main/java/org/apache/seata/server/config/SeataNamingserverWebConfig.java

```diff
@@ -33,7 +33,7 @@ public class SeataNamingserverWebConfig {
     public FilterRegistrationBean<RaftRequestFilter> raftRequestServletFilter(RaftRequestFilter raftRequestFilter) {
         FilterRegistrationBean<RaftRequestFilter> registrationBean = new FilterRegistrationBean<>();
         registrationBean.setFilter(raftRequestFilter);
-        registrationBean.addUrlPatterns("/api/v1/console/*");
+        registrationBean.addUrlPatterns("/api/v1/console/*", "/vgroup/v1/*");
```

Comment:
Are these two paths need the same filter function?

Reply:
> Are these two paths need the same filter function?

Yes, because the newly added transaction group must go through the leader node.